spec/models/concerns/material_spec.rb

```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
-
-shared_examples "material" do
-  it { is_expected.to have_one(:tube) }
-end
```

Comment:
why get rid of this test?

Reply:
This is a shared example and is a duplicate of spec/support/material.rb. I was getting a redefinition warning.